README.adoc

```diff
@@ -1,3 +1,10 @@
+https://maven-badges.herokuapp.com/maven-central/com.github.mizool/mizool/[image:https://maven-badges.herokuapp.com/maven-central/com.github.mizool/mizool/badge.svg?style=plastic[mizool
```

Comment:
leaving out the `?style=plastic` seems to work fine, and results in a closer match to the sonarcloud badge.